src/lib.rs

```diff
@@ -13,6 +13,74 @@ mod util;
 
 pg_module_magic!();
 
+/// TODO: presently, we deliberate on how to structure SQL-only tests.
+/// As soon as we agree on something these tests should be moved over.
+
```

Comment:
can we add a test that inserts the same value with two different keys, and a test with two values for the same key?

Reply:
done